src/System.IO.Ports/tests/Legacy/SerialPort/ReadTimeout.cs

```diff
@@ -36,146 +36,167 @@ private enum ThrowAt { Set, Open };
 
     #region Test Cases
 
+    [ActiveIssue(15961)]
     [ConditionalFact(nameof(HasLoopbackOrNullModem))]
     public void ReadTimeout_Default_Read_byte_int_int()
```

Comment:
I'm afraid It's just fluke that this problem has occurred on tests with 'timeout' in the name - lots of the tests will be sensitive to pauses on the CI machines, even if they're not really about timeouts per se.